app/models/email_template.rb

```diff
@@ -42,6 +43,8 @@ def users
       User.mentee.where.not(registration_token: nil)
     when :unregistered_mentors
       User.mentor.where.not(registration_token: nil)
+    when :daily_summary
+      User
```

Comment:
Is this correct ? It will be send to all users. Where will i call the query functions which return tasks ?